tests/zquantum/core/estimation/estimation_test.py

```diff
@@ -1,55 +1,26 @@
+from functools import partial
+
 import numpy as np
 import pytest
 import sympy
 from openfermion import IsingOperator, QubitOperator, qubit_operator_sparse
-from pyquil import Program
-from pyquil.gates import RX, RY, RZ, X, Y
-from functools import partial
-from zquantum.core import estimation
-from zquantum.core.circuit import Circuit, Qubit, Gate
-from zquantum.core.interfaces.mock_objects import (
-    MockQuantumBackend,
-    MockQuantumSimulator,
-)
+from zquantum.core.estimation import (allocate_shots_proportionally,
```

Comment:
What kind of formatter did you use? Running `black` on my side resulted in newlines between the `(` and `)`

Reply:
This was my first commit with pre-commit hooks configured, perhaps something is wrong with my configuration –will be investigating later.